### DISCUSSION TITLE
Can  kafka topic name be configured manually during helm deployment?

### DISCUSSION THREAD

**Discussion by wallyell** [milvus-io/milvus#26164] [Q&A and General discussion]
I can't find the kafka topic name  in the config file.
the topic in kafka will be created automatically with these three names below:

- by-dev-datacoord-timetick-channel
- by-dev-rootcoord-delta_0
- by-dev-rootcoord-dml_0

and producer just wrote one partition during work.

The second problem is the message can't be read directly because of messy code.

**Comment by yhmo** [comment]
In v2.3, we allow user pre-define topic names in milvus.yaml:
https://github.com/milvus-io/milvus/blob/4e1b65d38f7a51762a56b01e15ef0fb32b0e4a84/configs/milvus.yaml#L481C18-L481C18

In v2.2.x, topic names are automatically generated, not allowed to pre-define.

**Reply by wallyell** [reply]
The v2.3 version will be released in this week?

**Reply by yhmo** [reply]
If everything is ok. Next Monday.
